crypto/curve25519/curve25519.c

```diff
@@ -279,6 +278,12 @@ static void ed25519_sign_s2n_bignum(
   abort();
 }
 
+static int ed25519_verify_s2n_bignum(uint8_t R_have_encoded[32],
```

Comment:
why do you call it `_have_encoded`?

Reply:
It's meant to mean that this is the value that we have computed, distinguishing from the value that were given (i.e. `R_expected`).